python/pyspark/pandas/indexes/base.py

```diff
@@ -1141,10 +1141,21 @@ def is_type_compatible(self, kind: str) -> bool:
         """
         return kind == self.inferred_type
 
-    def dropna(self) -> "Index":
+    def dropna(self, how: str = "any") -> "Index":
         """
         Return Index or MultiIndex without NA/NaN values
 
+        Parameters
+        ----------
+        how : {'any', 'all'}, default 'any'
+            If the Index is a MultiIndex, drop the value when any or all levels
+            are NaN.
+
+
```

Comment:
nit: we usually have just one line here between the items ?

Reply:
Good catch!